tests/testthat/_snaps/verb-slice.md

```diff
@@ -24,6 +24,51 @@
       ! `slice_tail()` is not supported on database backends.
       i Please use `slice_max()` instead.
 
+# min, max, and sample generate useful sql
+
+    Code
+      slice_min(lf, x, n = 1)
```

Comment:
not related to this PR:

should it be discouraged to use `|>` in snapshot code? Since it doesn't show up in the code here, it makes it slightly more toilsome to track input+output when reviewing.